src/components/NewBook.js

```diff
@@ -23,7 +22,10 @@ const NewBook = () => {
         <input
           type="text"
           placeholder="Book title"
-          onChange={(e) => setNewBook({ ...newBook, title: e.target.value })}
+          onChange={(e) => setNewBook({
+            ...newBook,
+            title: e.target.value,
+          })}
         />
         <select
           name="author"
```

Comment:
- [Optional] For great UI/UX, I think it will be better if the color of these text can be bolder. ℹ️

Reply:
Well Noted!